GVRf/Framework/jni/engine/importer/assimp_importer.cpp

```diff
@@ -68,4 +68,428 @@ Mesh* AssimpImporter::getMesh(int index) {
 
     return mesh;
 }
+
+bool AssimpImporter::create_instance(JNIEnv *env, const char* class_name,
+        jobject& new_instance) {
```

Comment:
I think this is fine. These calls aren't used from Java native method. They are just using env for other purposes like calling back to Java. As you can see similar cases also happen with textures classes